packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -744,4 +744,40 @@ MeetingUtil.handleDeviceLogging = (devices = []) => {
   });
 };
 
+MeetingUtil.endMeetingForAll = (meeting, options = {}) => {
+  if (meeting.meetingState === FULL_STATE.INACTIVE) {
+    // TODO: clean up if the meeting is already inactive
+    return Promise.reject(new MeetingNotActiveError());
+  }
+
+  const defaultOptions = {
+    locusUrl: meeting.locusUrl
+  };
+
+  const leaveOptions = {...defaultOptions, ...options};
+
+  return meeting.meetingRequest
+    .endMeetingForAll(leaveOptions)
+    .then((response) => Promise.resolve(response))
```

Comment:
this line is useless - doesn't do anything. The leave() method does a call to `onFullLocus()` here. Why are we not doing it here too?

Reply:
+1

Reply:
Removed the line and added onFullLocus()